named.go

```diff
@@ -151,7 +151,7 @@ func bindArgs(names []string, arg interface{}, m *reflectx.Mapper) ([]interface{
 	fields := m.TraversalsByName(v.Type(), names)
 	for i, t := range fields {
 		if len(t) == 0 {
-			return arglist, fmt.Errorf("could not find name %s in %#v", names[i], arg)
+			return arglist, fmt.Errorf("could not find the field named \"%s\" in \"%#v\"", names[i], arg)
```

Comment:
You can use `%q` here instead of quoting the verbs yourself, and/or use raw strings (backticks) to avoid having to escape the quotes.